modules/ding_contact/ding_contact.strongarm.inc

```diff
@@ -15,8 +15,6 @@ function ding_contact_strongarm() {
   $strongarm->api_version = 1;
   $strongarm->name = 'antibot_form_ids';
   $strongarm->value = 'comment_node_*
-user_login
-user_pass
```

Comment:
As far as I can tell updating these values will not take effect without also reverting the feature. Please add that to an update hook.